mappings/net/minecraft/world/WorldType.mapping

```diff
@@ -1,19 +1,20 @@
-CLASS none/auf net/minecraft/world/DimensionType
+CLASS none/auf net/minecraft/world/WorldType
 	FIELD a OVERWORLD Lnone/auf;
 	FIELD b NETHER Lnone/auf;
 	FIELD c END Lnone/auf;
 	FIELD d id I
 	FIELD e name Ljava/lang/String;
-	FIELD f worldName Ljava/lang/String;
+	FIELD f suffix Ljava/lang/String;
 	FIELD g providerClass Ljava/lang/Class;
 	METHOD <init> (Ljava/lang/String;IILjava/lang/String;Ljava/lang/String;Ljava/lang/Class;)V
 		ARG 0 id
 		ARG 1 name
-		ARG 2 worldName
+		ARG 2 suffix
 		ARG 3 providerClass
 	METHOD a getID ()I
-	METHOD a getDimensionByID (I)Lnone/auf;
+	METHOD a getById (I)Lnone/auf;
 		ARG 0 id
 	METHOD b getName ()Ljava/lang/String;
-	METHOD c getWorldName ()Ljava/lang/String;
-	METHOD d constructDimension ()Lnone/aue;
+	METHOD c getSuffix ()Ljava/lang/String;
+	METHOD d construct ()Lnone/aue;
```

Comment:
It creates a new instance of the `WorldProvider` assigned to that `WorldType` through reflection. What is that usually called? Should it just be `createProvider`?